### DISCUSSION TITLE
Is there a way to redirect logging data to the text control?

### DISCUSSION THREAD

**Discussion by mikisama** [flet-dev/flet#4205] [Q&A]
### Question

Is there a way to redirect logging data to the text control?

```py
import logging
logging.basicConfig(level=logging.INFO)

# After config the logging, I want to display this text in a flet text control.
logging.info("this is a info")
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ap4499** [comment]
I think so.. Flet supports async apps pretty well, check this out: https://flet.dev/docs/getting-started/async-apps/.

The Threading example of the timer is what I used for a similar scenario that I had. 

As for the underlying architecture, to replicate a terminal in your app I'd do this:
- Continually read the logging file, and for each new line in the logging file, create a new flet Text object. Place (and add) all of those Flet text objects within a scrollable container (using a Column).
